### PR TITLE
feat: Remove Squidex from CRN server app

### PR DESCRIPTION
**Pull request by peterstarling**
- remove data providers
- remove some parsers
- removal part 1
- removal part 2
- update events

### CODE REVIEW

File: apps/crn-server/src/data-providers/transformers/event.ts [hunk 1]
```diff
@@ -1,16 +1,4 @@
...
-} from '@asap-hub/model';
-
-import {
-  EventContentFragment,
-  ExternalAuthors,
-  ExternalAuthorsFlatDataDto,
-} from '../../autogenerated-gql/graphql';
-import { createUrl } from '../../utils/urls';
 
 export const getMeetingMaterial = <T>(
```
Comment (gabiayako) [inline apps/crn-server/src/data-providers/transformers/event.ts:3]: I believe this `getMeetingMaterial` is not being used as well

File: apps/crn-server/test/fixtures/discover.fixtures.ts [hunk 2]
```diff
@@ -3,117 +3,8 @@ import {
...
-
-export const getSquidexGraphqlDiscover = () => ({
-  id: 'ec3086d4-aa64-4f30-a0f7-5c5b95ffbcca',
-  created: '2020-09-23T16:34:26.842Z',
-  lastModified: '2021-05-14T14:48:46Z',
-  version: 42,
-  flatData: squidexGraphqlDiscoverFlatData(),
-});
-
 const getSquidexGraphqlDiscoverMembersResponse = (
```
Comment (gabiayako) [inline apps/crn-server/test/fixtures/discover.fixtures.ts:8]: Maybe we could rename this?

```suggestion
const getGraphqlDiscoverMembersResponse = (
```

File: apps/crn-server/test/fixtures/events.fixtures.ts [hunk 3]
```diff
@@ -500,14 +484,6 @@ export const getListEventResponse = (
...
 
-export const getRestEvent = (): RestEvent => ({
-  id: 'squidex-event-id',
-  created: '2021-02-23T19:32:00Z',
-  lastModified: '2021-02-23T19:32:00Z',
-  version: 42,
-  data: getEventInput(),
-});
-
 export const getEventInput = () => ({
```
Comment (gabiayako) [inline apps/crn-server/test/fixtures/events.fixtures.ts:487]: I believe we can delete this

File: apps/crn-server/test/fixtures/interest-groups.fixtures.ts [hunk 4]
```diff
@@ -1,24 +1,19 @@
...
 import { appName, baseUrl } from '../../src/config';
-import { InterestGroupPayload } from '../../src/handlers/event-bus';
 import { createEventBridgeEventMock } from '../helpers/events';
-import {
-  getSquidexGraphqlTeam,
-  getContentfulGraphqlTeam,
-} from './teams.fixtures';
+import { getContentfulGraphqlTeam } from './teams.fixtures';
 
 export const queryInterestGroupsResponse = {
```
Comment (gabiayako) [inline apps/crn-server/test/fixtures/interest-groups.fixtures.ts:18]: I believe we can delete this `queryInterestGroupsResponse` as well

File: apps/crn-server/test/fixtures/labs.fixtures.ts [hunk 5]
```diff
@@ -59,15 +98,6 @@ export const getSquidexGraphqlLab = () => ({
   id: '12345',
```
Comment (gabiayako) [inline apps/crn-server/test/fixtures/labs.fixtures.ts:98]: this `getSquidexGraphqlLab` also can be deleted

File: apps/crn-server/test/fixtures/reminders.fixtures.ts
Comment (gabiayako) [inline apps/crn-server/test/fixtures/reminders.fixtures.ts:1]: I believe we can delete `getReferencingWorkingGroupsContents` from here

File: apps/crn-server/test/helpers/users.ts [hunk 6]
```diff
@@ -1,43 +1,7 @@
...
-
-  const user = Object.entries(userData).reduce((acc, [key, value]) => {
-    acc[key] = { iv: value };
-    return acc;
-  }, {} as { [key: string]: { iv: unknown } });
-
-  return users.create(user as RestUser['data']);
-};
-
 export const createRandomOrcid = () =>
```
Comment (gabiayako) [inline apps/crn-server/test/helpers/users.ts:5]: is this being used anywhere?

**Review by gabiayako — APPROVED** [review]

File: apps/crn-server/test/helpers/users.ts (hunk 6 above)
Comment (peterstarling) [reply apps/crn-server/test/helpers/users.ts:5]: it is not but i think we might leave it, this could POTENTIALLY be used soon so

**Pull request merged**